packages/botonic-react/src/utils.js

```diff
@@ -92,3 +97,10 @@ export function renderComponent({ renderBrowser, renderNode }) {
   else if (isNode()) return renderNode()
   throw new Error('Unexpected process type. Not recognized as browser nor node')
 }
+
+export const getWebchatElement = () =>
+  document.getElementById('botonic-webchat')
```

Comment:
Beware that this will cause problems if somebody has 2 instances of our webchat in the same page. A solution could be to generate a unique id for each webchat instance.

I don't think this will be a common case, but I leave the note here to keep that in mind for the future. Maybe we can add a comment in the documentation.

Reply:
You're right! Maybe we will need to do the same for the other "important" id `botonic-scrollable-content`. I'm adding a pair of todos there 👍